### PR TITLE
Fix the ansible stderr issue

### LINKED ISSUE #54: Ansible stderr often gets swallowed
When runs of ansible-playbook error, they often result in none of their own stderr being output. Instead, I typically get a Python stack trace from inside of teflo. This then requires me to run the playbook again, manually, in order to see the output error.

Please don't swallow stderr

### PR DESCRIPTION
**Pull request by JunqiZhang0**
Fix #54

### CODE REVIEW

**Comment by JunqiZhang0** [comment]
@greg-hellings , this fix will output the original error from ansible, you can test it out, if there's other requirements, let me know

File: teflo/helpers.py [hunk 1]
```diff
@@ -563,7 +563,8 @@ def exec_local_cmd_pipe(cmd, logger, env_var=None):
             logger.info(output.strip())
     rc = proc.poll()
     if rc != 0:
-        error = proc.stderr.readline().decode('utf-8')
+        for line in proc.stderr:
+            error = error + proc.stderr.readline().decode('utf-8')
```
Comment (rujutashinde) [inline teflo/helpers.py:567]: Do we use the variable `line` here ?

Comment (JunqiZhang0) [reply teflo/helpers.py]: done

**Review by rujutashinde — APPROVED** [review]
LGTM

**Pull request merged**